src/lib.rs

```diff
@@ -5774,6 +5774,11 @@ unsafe impl<'a> SplitByteSlice for RefMut<'a, [u8]> {
     }
 }
 
+// TODO(#429): Add a "SAFETY" comment and remove this `allow`.
```

Comment:
Can you add safety comments here and below? Our policy is not to introduce any *new* undocumented `unsafe` code while we burn down the existing TODOs.